libs/application/core/src/lib/messages.ts

```diff
@@ -208,7 +213,7 @@ export const coreErrorMessages = defineMessages({
     description: 'Oops! Could not assign to the application',
   },
   couldNotAssignApplicationErrorDescription: {
-    id: 'application.system:could.not.assign.application.error.title',
+    id: 'application.system:could.not.assign.application.error.description',
```

Comment:
Unrelated to my changes but had to be included since the id of the message was a duplicate of the one above which blocked being able to sync changes to Contentful.